### ISSUE TITLE
Failed to download forest germline.v0.7.0.forest.gz

### ISSUE THREAD

**Issue by satzel** [luntergroup/octopus#124]
**Describe the bug**
Hiya, I tried to install the option to run filtering using random forest but can't download germline.v0.7.0.forest.gz or somatic.v0.7.0.forest.gz
When checking the google bucket only version 0.6.0. and 0.6.3 seem to be there

**Version**

```shell
$ octopus --version
// PASTE OUTPUT HERE
```
octopus version 0.7.0 (develop 595f8917)
Target: x86_64 Linux 3.10.0-957.10.1.el7.x86_64
SIMD extension: AVX512
Compiler: GNU 9.2.0
Boost: 1_72

**Command**
Command line to install octopus:
```shell
$ ./scripts/install.py --forests --htslib /apps/eb/skylake/software/HTSlib/1.9-foss-2018b
```

**Comment by dancooke** [comment]
Hey, unfortunately the v0.7.0 forests are not available just yet - I'll make them available once v0.7.0 is a tagged release. Note that the earlier forest versions won't work with the latest develop build of Octopus, so your options are either to just use. v0.6.3b, or train your own forest.

**Comment by dancooke** [comment]
Duplicate of #103.

**Comment by satzel** [comment]
is it easy enough to down grade to v0.6.3b? I can search myself, unless there is a super quick pointer you can post?
thanks alot :)

**Comment by dancooke** [comment]
You can either checkout `release/v0.6.3-beta`, or download the tarball [here](https://github.com/luntergroup/octopus/archive/v0.6.3-beta.tar.gz), or install via [Bioconda](https://anaconda.org/bioconda/octopus).

**Comment by satzel** [comment]
Ok I got octopus v0.6.3-beta (release/0.6.3-beta 8b0e2968) and downloaded the two respective forest files from the goolgle bucket and when I run this command:

  /octopus.v.6.3/octopus/bin/octopus -R  ref/Homo_sapiens_assembly38.fasta -I blood_01_recal.bam\
     -C cancer \
     --forest /octopus.v.6.3/octopus/resources/forests/germline.v0.6.3-beta.forest \
     --somatic-forest /octopus.v.6.3/octopus/resources/forests/somatic.v0.6.3-beta.forest\
     --debug \
     -o filtered.octopus_both.vcf.gz

I get:
2020-05-21 09:07:04] <DEBG> Encountered an error whilst calling, attempting to cleanup
[2020-05-21 09:07:04] <INFO> Removed 2 temporary files
[2020-05-21 09:07:04] <INFO>                            -             100%            9m 4s                 -
[2020-05-21 09:07:04] <EROR> A program error has occurred:
[2020-05-21 09:07:04] <EROR> 
[2020-05-21 09:07:04] <EROR>     Encountered an exception during calling 'std::bad_alloc'. This means
[2020-05-21 09:07:04] <EROR>     there is a bug and your results are untrustworthy.
[2020-05-21 09:07:04] <EROR> 
[2020-05-21 09:07:04] <EROR> To help resolve this error run in debug mode and send the log file to
[2020-05-21 09:07:04] <EROR> https://github.com/luntergroup/octopus/issues.
[2020-05-21 09:07:04] <INFO> ------------------------------------------------------------------------

I ran in debug mode and got the log file but its 70MB, shall I uplaod it here? 

I could run variant calling in cancer mode on the same file in v0.7.0 without errors but then I couldn't add the forests.

Do you have advise for this?
Many thanks :)

**Comment by dancooke** [comment]
Likely that this is a bug in v0.6.3b that's been fixed in v0.7.0. Unfortunately, you'll need to wait until v0.7.0 is released to access the new forests. Shouldn't be too much longer now!
